myTinyURL/convert/convert.py

```diff
@@ -4,26 +4,60 @@
 import random, string
 
 class Convert:
+    """This class is the implementation for url conversion
+    """
+
+    # the length of short_id
     CONST_LENGTH = 6
-    def long2short(httpurl):
-        short_id = Convert.gen()
-        record = URL(httpurl=httpurl, short_id=short_id)
+
+    def long2short(http_url):
```

Comment:
Let's try to understand the type of http_url.
In views, when get url, print type(url).